python/tox.ini

```diff
@@ -1,5 +1,5 @@
 [tox]
-envlist = py{27,35},py{36,37,38,39}-{sbp2json,noextra}
+envlist = py{36,37,38,39}-{sbp2json,noextra}
```

Comment:
What work do we need to do re-enable tests for Python 2.7 and Python 3.5?  It would be nice to keep those for the time being.

Reply:
I need to move the new tool out of `tests`, or fiddle with `sbpg` so that it is py27 compatible.  c.f. https://github.com/swift-nav/libsbp/pull/969#discussion_r628288755 about where to put this new tool.

Reply:
I'm find to fork this to a new PR/Jira if that helps move things forward

Reply:
Moved to https://github.com/swift-nav/libsbp/pull/979 , and moved it into `generator/` to avoid the py27 problem.